app/views/cases/shared/_closed_case_details.html.slim

```diff
@@ -34,6 +34,16 @@ tbody.response-details
       th = t('common.case.outcome')
       td = case_details.outcome.name
 
+  - if case_details.team_responsible_for_outcome.present?
+    tr.team-responsible
+      th = t('common.case.team_responsible_for_outcome')
+      td = case_details.team_responsible_for_outcome.name
+
+  - if case_details.outcome_reasons.present?
+    tr.outcome-reasons
+      th = t('common.case.outcome_reason')
+      td = case_details.pretty_outcome_reasons
+
```

Comment:
Just curious,  are the two fields editable?  if do,  are you going to add the function later?

Reply:
They are editable using the "Edit case details" and "Edit closure details" links, but not using the individual fields as it the case with OffSars / OffSar Complaints - is this what you mean?

Reply:
my doubt is caused by not seeing any changes on edit_case_details from your PR :),  but I think as long as it is editable at one of them e..g Edit closure details,  it would be fine

Reply:
I have another PR almost ready for review on this! :)